src/main/webapp/statistics.html

```diff
@@ -7,22 +7,42 @@
     <h1>Statistics</h1>
     <script type="text/javascript"
     src="https://www.gstatic.com/charts/loader.js"></script>
+    <h2>General</h2>
+    <ul>
+      <li>19% of all pregnancies end in abortion</li>
+      <li>1 in 3 American women will have an abortion by age 45</li>
+      <li>6 in 10 women having abortions already have a child</li>
+    </ul>
     <script type="text/javascript">
 
     //load Charts and the corechart package.
     google.charts.load('current', {'packages': ['corechart']});
 
+    //Draw the pie chart  for racial distribution of abortions
+    google.charts.setOnLoadCallback(drawRaceChart);
+
     //Draw the pie chart for when abortions typically occur
     google.charts.setOnLoadCallback(drawWhenChart);
 
-    //Draw the column chart for the cost of an abortion
-    google.charts.setOnLoadCallback(drawCostChart);
-    //Draw the column chart for the length of an abortion
-    google.charts.setOnLoadCallback(drawLengthChart);
-
     //draw the ___ chart for types of procedures based on weeks of pregnancy
     //google.charts.setOnLoadCallback(drawProcedureChart);
 
+    function drawRaceChart() {
+      var raceData = google.visualization.arrayToDataTable([
+        ['Ethnic background' , 'Percentage'],
+        ['White',39],
+        ['Black',28],
```

Comment:
Add space after comma and not before ['White', 39]

Reply:
implemented check